### PR TITLE
OCPBUGS10029: correct cluster access modules MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-10029

Link to docs preview:
https://58067--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#accessing-microshift-cluster_microshift-install-rpm

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/microshift-accessing-cluster-open-firewall.adoc [hunk 1]
```diff
@@ -3,10 +3,10 @@
 // microshift/microshift-install-rpm.adoc
 
 :_content-type: PROCEDURE
-[id="accessing-microshift-cluster-remotely-as-admin_{context}"]
-= Accessing the {product-title} cluster remotely as admin
+[id="microshift-accessing-cluster-remotely-open-firewall_{context}"]
+= Opening the firewall for remote access the {product-title} cluster
```
Comment (dhellmann) [inline modules/microshift-accessing-cluster-open-firewall.adoc:7]:
```suggestion
= Opening the firewall for remote access to the {product-title} cluster
```

File: modules/microshift-accessing-cluster-remotely.adoc [hunk 2]
```diff
@@ -29,17 +29,17 @@ Use the following procedure to access the {product-title} cluster from a remote
...
-[user@microshift]$ MICROSHIFT_MACHINE=<name or IP address of {product-title} machine>
+[user@workstation]$ MICROSHIFT_MACHINE=<name or IP address of {product-title} machine>
 ----
 
-. As `user@workstation`, copy the generated `kubeconfig` file from the {op-system} machine running {product-title} to your local machine by running the following command:
+. As `user@workstation`, copy the generated `kubeconfig` file that contains the host name or IP address you want to connect with from the {op-system} machine running {product-title} to your local machine by running the following command:
 +
 [source,terminal]
 ----
 [user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig" > ~/.kube/config
```
Comment (dhellmann) [inline modules/microshift-accessing-cluster-remotely.adoc:39]: This command needs to point to the kubeconfig file with the info for accessing the cluster remotely. The file it refers to now is the one for local access.

```suggestion
[user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/$MICROSHIFT_MACHINE/kubeconfig" > ~/.kube/config
```

**Review by dhellmann — COMMENTED** [review]
This is a good improvement. I noticed 1 typo and had one technical note.

**Comment by ShaunaDiaz** [comment]
@dhellmann Are you ready to LGTM? If so, I just need QE and we can ship this one off.

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Comment by dhensel-rh** [comment]
I noticed a reference to RHEL 8 under Installing MicroShift  from an RPM Package

Preparing to install MicroShift from an RPM package --> Procedure

 Under New Red Hat Enterprise Linux 8.x Installation, select Click here to create them automatically.

I thought 9.2 was a system Requirement

**Comment by dhensel-rh** [comment]
Under **Accessing the MicroShift cluster remotely** Is Step 5. still relevant ? If you are using the correct kubeconfig in Step 4, you should already have the MICROSHIFT_MACHINE in the  ~/.kube/config file.

**Comment by ShaunaDiaz** [comment]
> I noticed a reference to RHEL 8 under Installing MicroShift from an RPM Package
> 
> Preparing to install MicroShift from an RPM package --> Procedure
> 
> Under New Red Hat Enterprise Linux 8.x Installation, select Click here to create them automatically.
> 
> I thought 9.2 was a system Requirement

@dhensel-rh I looked through the modules and searched the repo, and did not find this reference. Can you mark the line for me or send me a link?

**Comment by ShaunaDiaz** [comment]
> Under **Accessing the MicroShift cluster remotely** Is Step 5. still relevant ? If you are using the correct kubeconfig in Step 3, you should already have the MICROSHIFT_MACHINE in the ~/.kube/config file.

@dhellmann Can we remove this step? Seems like we can, or we need more explanation.

**Comment by dhellmann** [comment]
> > Under **Accessing the MicroShift cluster remotely** Is Step 5. still relevant ? If you are using the correct kubeconfig in Step 3, you should already have the MICROSHIFT_MACHINE in the ~/.kube/config file.
> 
> @dhellmann Can we remove this step? Seems like we can, or we need more explanation.

Yes, good catch @dhensel-rh.

In modules/microshift-accessing-cluster-remotely.adoc lines 50-56 can be removed.

**Comment by dhensel-rh** [comment]
> > I noticed a reference to RHEL 8 under Installing MicroShift from an RPM Package
> > Preparing to install MicroShift from an RPM package --> Procedure
> > Under New Red Hat Enterprise Linux 8.x Installation, select Click here to create them automatically.
> > I thought 9.2 was a system Requirement
> 
> @dhensel-rh I looked through the modules and searched the repo, and did not find this reference. Can you mark the line for me or send me a link?

@ShaunaDiaz Here is a link to the document in question. Apologies if it is outside the scope of this PR. I'm still learning how to navigate 

https://58067--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#microshift-install-rpm-preparing_microshift-install-rpm

**Comment by ShaunaDiaz** [comment]
1349789 Update modules/microshift-accessing-cluster-remotely.adoc and modules/microshift-install-preparing-rpm.adoc
c5cd8ac Rebased

File: modules/microshift-accessing-cluster-remotely.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+[source,terminal]
+----
+[user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/$MICROSHIFT_MACHINE/kubeconfig" > ~/.kube/config
+----
+
+. Update the permissions on your `~/.kube/config` file by running the following command:
++
+[source,terminal]
+----
+$ chmod go-r ~/.kube/config
```
Comment (sheriff-rh) [inline modules/microshift-accessing-cluster-remotely.adoc:47]: Is this command also run as the `[user@workstation]`? If I understand correctly, the ssh command in the previous step is writing to the `~/.kube/config` file on the `[user@workstation]` machine. 

Can we specify which user on which machine for this step as you have done with the other steps in this procedure?

**Review by sheriff-rh — COMMENTED** [review]
Looks great! One small question.

**Comment by jboxman-rh** [comment]
@ShaunaDiaz, looks like this one doesn't fit cleanly into 4.12.

**Comment by ShaunaDiaz** [comment]
@jboxman-rh Hmm, I can do a manual CP unless you want to look into it further?

**Pull request merged**